### ISSUE TITLE
Is this repo still active?

### ISSUE THREAD

**Issue by nitrocode** [pghalliday-cookbooks/sonarqube#36]
Hi @willmanio. Thank you so much for your work on this repo. I was just wondering is this repo still active? There's a number of open PRs and I use your chef cookbook. I was planning to fork it and merge all the PRs to get the latest one but thought I'd ping you first. If it's maintained here it's even easier for me. 

Please let me know!

Thank you and happy new year.

**Comment by willmanio** [comment]
Hi! Thanks for reaching out. Truth be told, I haven’t had much time to contribute to OSS in 2017 and 2018, and I regretfully have let this repository linger. I apologize for that.

Would you be interested in becoming the owner? I am happy to transfer this repository to someone who will have more time to dedicate to the community.

**Comment by farrukhnajmi** [comment]
@willmanio I am interested in becoming the owner. I have used this recipe and many others to provision project specific dev servers in a consulting company where each time we have a new project start we need the same set of dev tools provisioned on that server. I have many years of open source background though not so much in recent years. I am not a ruby expert but am a quick learner and have been learning.

**Comment by willmanio** [comment]
Hi @farrukhnajmi. I opened a transfer request for this repository to your account. Once you accept the request, you should become the new owner. ⭐️

**Comment by farrukhnajmi** [comment]
Thank you. Would it be ok if I contact you occasionally if I have a question? Some form of IM would be great. BTW, It may be better for you to stay with the project for an interim period and simply add me as an admin in addition to your self.

**Comment by willmanio** [comment]
Of course, you can contact me any time. My email address is on my GH profile. I've added you as an admin; feel free to merge PRs. Let's plan on transferring ownership to you within the next month or two.

**Comment by jblaine** [comment]
Hi all - doesn't look like this went anywhere yet. Is that accurate?

**Comment by willmanio** [comment]
Hi @jblaine, indeed I haven't heard from @farrukhnajmi in a few months. I'm finding that I'm beginning to have some free time once again. If there is still interest, I can begin whipping this repo back into shape over the next few weeks.

**Comment by farrukhnajmi** [comment]
Hi guys,

I am very sorry that I have been unable to step up to help out. Due to
changes at my current job I do not see this changing in the near future.
Again I am very sorry for doing the ball.

**Comment by willmanio** [comment]
@farrukhnajmi not a problem at all! My schedule is freeing a bit, so I can dive into the outstanding issues and PRs.
